content/docs/start/list-of-operations.mdx

```diff
@@ -602,7 +602,7 @@ Possible errors:
 | REVOKE_SPONSORSHIP_ONLY_TRANSFERABLE | -4 | Sponsorship cannot be removed from this ledgerEntry. This error will happen if the user tries to remove the sponsorship from a ClaimableBalanceEntry. |
 
 ## Set Trustline Flags
-[JavaScript](http://stellar.github.io/js-stellar-sdk/Operation.html#.setTrustLineFlags) | [Java](http://stellar.github.io/java-stellar-sdk/org/stellar/sdk/SetTrustLineFlags.Builder.html) | [Go](https://godoc.org/github.com/stellar/go/txnbuild#SetTrustLineFlags)
+[JavaScript](http://stellar.github.io/js-stellar-sdk/Operation.html#.setTrustLineFlags) | [Java](http://stellar.github.io/java-stellar-sdk/org/stellar/sdk/SetTrustLineFlagsOperation.Builder.html) | [Go](https://godoc.org/github.com/stellar/go/txnbuild#SetTrustLineFlags)
```

Comment:
Good catch, but I believe the casing is wrong here. This link 404s:

http://stellar.github.io/java-stellar-sdk/org/stellar/sdk/SetTrustLineFlagsOperation.Builder.html

while this link resolves (note TrustLine vs Trustline):

https://stellar.github.io/java-stellar-sdk/org/stellar/sdk/SetTrustlineFlagsOperation.Builder.html

Reply:
Awwww, man! I totally didn't see that when looking in the Java docs. Thanks!

It appears most other SDKs and the Stellar docs in general use the TrustLine casing. Is that generally preferred? Is there a specific reason the Java SDK uses the lowercase L?